portal/portal-impl/impl/src/java/org/sakaiproject/portal/charon/site/PortalSiteHelperImpl.java

```diff
@@ -306,16 +309,16 @@ private Map<String, Object> getSiteMap(Site site, String currentSiteId, String u
 
 		Map<String, Object> siteMap = new HashMap<>();
 		siteMap.put("id", site.getId());
-		siteMap.put("title", site.getTitle());
+		siteMap.put("title", site.getId().equals("~admin") ? rb.getString("sit_mywor_admin") : site.getId().equals("!admin") ? rb.getString("sit_admin") : userId != null && site.getId().equals(siteService.getUserSiteId(userId)) ? rb.getString("sit_mywor") : site.getTitle());
```

Comment:
Hi @st-manu , i have tested locally everything works great now. Thanks. just one small question, do you think its a good idea to put all the cases in a single line, its hard to read